tests/test_all_fields.py

```diff
@@ -20,7 +57,7 @@ def test_run(self):
         - Verify that more than just the automatic fields are replicated for each stream.
         """
 
-        expected_streams = self.expected_streams()
+        expected_streams = self.expected_streams() - {'ad_analytics_by_creative', 'ad_analytics_by_campaign'}
```

Comment:
Add a comment about the above streams removal from expected_streams.

Reply:
We are not going to skip these streams.